addons/portal/static/src/scss/portal.scss

```diff
@@ -23,6 +23,11 @@ body {
     // Set frontend direction that will be flipped with
     // rtlcss for right-to-left text direction.
     direction: ltr;
+    .o_skip_to_content {
+        // We require the z-index property to ensure that the "Skip to Content"
+        // button appears above the header.
+        z-index: $zindex-fixed + 1;
+    }
```

Comment:
Don't put this rule inside the `body` rule. Make selectors the smaller you can when you write css :+1: